experimental/packages/sdk-logs/src/export/ReadableLogRecord.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { IResource } from '@opentelemetry/resources';
+import type { Attributes, HrTime } from '@opentelemetry/api';
+import type { InstrumentationScope } from '@opentelemetry/core';
+import type { SeverityNumber } from '@opentelemetry/api-logs';
+
+export interface ReadableLogRecord {
+  readonly time: HrTime;
+  readonly traceId?: string;
+  readonly spanId?: string;
+  readonly traceFlags?: number;
```

Comment:
Should use `enum TraceFlags` here?

Reply:
According to the latest specification, context is now used. This has been changed to spanContext here